java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -31,46 +31,44 @@
 
 public class WorkspaceDriver {
 
-  private static File tmp;
-  private static Map<String, File> bazelVersions;
-  private static File runfileDirectory = new File(System.getenv("TEST_SRCDIR"));
+  private static Path tmp;
+  private static Map<String, Path> bazelVersions;
+  private static Path runfileDirectory = Paths.get(System.getenv("TEST_SRCDIR"));
 
-  private File currentBazel = null;
+  private Path currentBazel = null;
 
   /**
    * The current workspace.
    */
-  private File workspace = null;
+  private Path workspace = null;
 
   public static void setUpClass() throws IOException {
-    // Get tempdir
-    String _tmp = System.getenv("TEST_TMPDIR");
-    if (_tmp == null) {
-      File p = Files.createTempDirectory("e4b-tests").toFile();
-      p.deleteOnExit();
-      tmp = p;
+    String environmentTempDirectory = System.getenv("TEST_TMPDIR");
+    if (environmentTempDirectory == null) {
+      File tempDirectory = Files.createTempDirectory("e4b-tests").toFile();
```

Comment:
Really nit but you do toFile and then afterwards toPath. Maybe save the first and then you can skip the second transformation?

Reply:
right